wherehows-web/app/components/dataset-author.js

```diff
@@ -1,13 +1,6 @@
 import Ember from 'ember';
 
-const {
-  set,
-  get,
-  getWithDefault,
-  computed,
-  Component,
-  inject: { service }
-} = Ember;
+const { set, get, getWithDefault, computed, Component, inject: { service } } = Ember;
```

Comment:
Wouldn't it be easier to read and alter the properties if they're in separate lines?